src/seq/Seq.jl

```diff
@@ -138,8 +138,7 @@ export
     AA_Gap,
     FASTA,
     FASTASeqRecord,
-    FASTQReader,
-    FASTQWriter,
+    FASTQ,
     FASTQSeqRecord,
```

Comment:
I think it would be better to move these record types into deprecated.jl to deprecate them. I will do that in another pull request.